lib/CM5Receiver/CM5LeftConnector.tsx

```diff
@@ -0,0 +1,125 @@
+import type { ChipProps } from "@tscircuit/props"
+import { CM5Connector } from "./CM5Connector"
+
+const pinLabelsLeftSide = {
+  pin1: ["USB_OTG_ID"],
+  pin2: ["PCIe_CLK_nREQ"],
+  pin3: ["USB_N"],
+  pin4: ["PCIE_nWAKE"],
+  pin5: ["USB_P"],
+  pin6: ["PCIE_PWR_EN"],
+  pin7: ["GND_22"],
+  pin8: ["GND_23"],
+  pin9: ["PCIe_nRST"],
+  pin10: ["PCIe_CLK_P"],
+  pin11: ["VBUS_EN"],
+  pin12: ["PCIe_CLK_N"],
+  pin13: ["GND_24"],
+  pin14: ["GND_25"],
+  pin15: ["MIPI0_D0_N"],
+  pin16: ["PCIe_RX_P"],
+  pin17: ["MIPI0_D0_P"],
+  pin18: ["PCIe_RX_N"],
+  pin19: ["GND_26"],
+  pin20: ["GND_27"],
+  pin21: ["MIPI0_D1_N"],
+  pin22: ["PCIe_TX_P"],
+  pin23: ["MIPI0_D1_P"],
+  pin24: ["PCIe_TX_N"],
+  pin25: ["GND_28"],
+  pin26: ["GND_29"],
+  pin27: ["MIPI0_C_N"],
+  pin28: ["USB3_0_RX_N"],
+  pin29: ["MIPI0_C_P"],
+  pin30: ["USB3_0_RX_P"],
+  pin31: ["GND_30"],
+  pin32: ["GND_31"],
+  pin33: ["MIPI0_D2_N"],
+  pin34: ["USB3_0_DP"],
+  pin35: ["MIPI0_D2_P"],
+  pin36: ["USB3_0_DM"],
+  pin37: ["GND_32"],
+  pin38: ["GND_33"],
+  pin39: ["MIPI0_D3_N"],
+  pin40: ["USB3_0_TX_N"],
+  pin41: ["MIPI0_D3_P"],
+  pin42: ["USB3_0_TX_P"],
+  pin43: ["HDMI1_HOTPLUG"],
+  pin44: ["GND_34"],
+  pin45: ["HDMI1_SDA"],
+  pin46: ["HDMI1_TX2_P"],
+  pin47: ["HDMI1_SCL"],
+  pin48: ["HDMI1_TX2_N"],
+  pin49: ["HDMI1_CEC"],
+  pin50: ["GND_35"],
+  pin51: ["HDMI0_CEC"],
+  pin52: ["HDMI1_TX1_P"],
+  pin53: ["HDMI0_HOTPLUG"],
+  pin54: ["HDMI1_TX1_N"],
+  pin55: ["GND_36"],
+  pin56: ["GND_37"],
+  pin57: ["USB3_1_RX_N"],
+  pin58: ["HDMI1_TX0_P"],
+  pin59: ["USB3_1_RX_P"],
+  pin60: ["HDMI1_TX0_N"],
+  pin61: ["GND_38"],
+  pin62: ["GND_39"],
+  pin63: ["USB3_1_DP"],
+  pin64: ["HDMI1_CLK_P"],
+  pin65: ["USB3_1_DM"],
+  pin66: ["HDMI1_CLK_N"],
+  pin67: ["GND_40"],
+  pin68: ["GND_41"],
+  pin69: ["USB3_1_TX_N"],
+  pin70: ["HDMI0_TX2_P"],
+  pin71: ["USB3_1_TX_P"],
+  pin72: ["HDMI0_TX2_N"],
+  pin73: ["GND_42"],
+  pin74: ["GND_43"],
+  pin75: ["MIPI1_D0_N"],
+  pin76: ["HDMI0_TX1_P"],
+  pin77: ["MIPI1_D0_P"],
+  pin78: ["HDMI0_TX1_N"],
+  pin79: ["GND_44"],
+  pin80: ["GND_45"],
+  pin81: ["MIPI1_D1_N"],
+  pin82: ["HDMI0_TX0_P"],
+  pin83: ["MIPI1_D1_P"],
+  pin84: ["HDMI0_TX0_N"],
+  pin85: ["GND_46"],
+  pin86: ["GND_47"],
+  pin87: ["MIPI1_C_N"],
+  pin88: ["HDMI0_CLK_P"],
+  pin89: ["MIPI1_C_P"],
+  pin90: ["HDMI0_CLK_N"],
+  pin91: ["GND_48"],
+  pin92: ["GND_49"],
+  pin93: ["MIPI1_D2_N"],
+  pin94: ["MIPI1_D3_N"],
+  pin95: ["MIPI1_D2_P"],
+  pin96: ["MIPI1_D3_P"],
+  pin97: ["GND_50"],
+  pin98: ["GND_51"],
+  pin99: ["HDMI0_SDA"],
+  pin100: ["HDMI0_SCL"],
+}
+
+export type CM5LeftPinLabels =
+  (typeof pinLabelsLeftSide)[keyof typeof pinLabelsLeftSide][number]
+
+export const CM5LeftConnector = (props: ChipProps) => (
+  <CM5Connector
+    side="left"
+    name={`${props.name}_LEFT`}
```

Comment:
don't modify the name that is passed in, it's "not honest", better to modifu it in the caller (at the call site)